src/platforms/native/common/minidumps.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Minidump
+title: Minidumps
```

Comment:
This page contains two pieces of information:
- Briefly, what's a minidump in principle
- And then mostly, how to manually integrate minidumps without the SDK.

I'd suggest moving the majority of this file to "Guides", since it is a "custom" integration similar to Breakpad and Crashpad. We could consider leaving the part with "what's a minidump" here, but it might be easier to keep everything together.